locksmith/migrations/20220117151038-create-hook-notifier.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('HookNotifiers', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      network: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
+      sentKeyIds: {
```

Comment:
This will make it hard to get the last one processed...

Reply:
Let's keep track of a `type` (key or lock for now) and a objectId (that is either the key id or the lock id).